src/Integration.UnitTests/Helpers/ToolWindowServiceTests.cs

```diff
@@ -128,6 +129,27 @@ public void EnsureToolWindowExists_VsServiceCalled(int vsServiceResult)
             frameMock.Invocations.Should().BeEmpty();
         }
 
+        [TestMethod]
```

Comment:
Something about this is wrong I think. it works but I am not sure its supposed to.

The windowPane object at some point turns into a "ToolWindowPaneProxy" which is why the assertion does not test against the type "ToolWindowPane".